index.js

```diff
@@ -92,20 +92,11 @@ const getRealType = (value) => {
     const type = getType(value);
 
     if (type === 'object') {
-        if (value === null) {
-            return 'null';
-        }
-        return value.constructor.name.toLowerCase();
+        return Object.prototype.toString.call(value).slice(8, -1).toLowerCase();
     }
 
     if (type === 'number' && !Number.isFinite(value)) {
-        if (Number.isNaN(value)) {
-            return 'NaN';
-        }
-        if (value < 0) {
-            return '-Infinity';
-        }
-        return 'Infinity';
+        return value.toString();
```

Comment:
Лучше всё же явно вернуть строку, её потом можно найти поиском, глазами прочитать и всё такое.

Reply:
```suggestion
        return value.toString(); // 'NaN', 'Infinity', '-Infinity'
```
А так имеет право на жизнь?

Или так в любом случае лучше:
``` suggestion
        if (type === 'number') {
            if (Number.isNaN(value)) {
                return 'NaN';
            }
            if (value === -Infinity) {
                return 'Infinity';
            }
            if (value === Infinity) {
                return 'Infinity';        
            }
            return type;
        }
```